core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/iterator/GroupIterator.java

```diff
@@ -474,10 +477,14 @@ public MinAggregate(Min operator) {
 		@Override
 		public void processAggregate(BindingSet s) throws QueryEvaluationException {
 			Value v = evaluate(s);
+			boolean strict = false;
+			if (strategy instanceof StrictEvaluationStrategy) {
+				strict = true;
+			}
```

Comment:
Unfortunately we also have subclasses of `StrictEvaluationStrategy` that are not 'non strict' - like the FederatedEvaluationStrategy. 

I think a simple way to fix is to turn this around: set `strict` to `true` by default, and only if the strategy is an instance of `ExtendedEvaluationStrategy`, set it to false.